Automattic/wc-calypso-bridge
# Remove Legacy OBW

While I'm still not entirely certain _how_, the old OBW that was built for the ecommerce plan was still launching for new stores. This PR removes all the code around the legacy OBW + Setup Checklist, and instead favors the new default experience that ships in WooCommerce core.

The second commit in this PR adds in some `function_exists` checks to allow for the plugin to be installed "normally" alongside the bundled version in wpcomsh on a biz plan site, which allows for testing of this branch.

__To Test__
This one is a bit involved, but really the only way to truly test this out as I couldn't recreate the buggy state locally myself.

0. Give yourself at least 600 wpcom credits on an account you want to use to...
1. Add a new ecommerce plan test site. This can be done by using the "switch site" tool in the calypso sidebar, and selecting Add new site at the bottom.
2. Go through the checkout flow, use a .blog domain
3. Note in the final step when you click the setup button to launch your store, you are redirected to the new atomic site, and the old OBW is shown ( you can see this all in @pmcpinto's [video here](https://www.loom.com/share/f2ecfdbf95a14f999e101f046b8cc2ac)
4. manually change the URL to `/wp-admin` on your new site
5. Zip up this branch of wc-calypso-bridge and install + activate it on the site. Make sure the unzipped folder for the plugin is named `wc-calypso-bridge`.
6. navigate to `/wp-admin/admin.php?page=wc-setup&calypsoify=1`, note the woo core onboarding flow is shown.
